ftw/keywordwidget/configure.zcml

```diff
@@ -83,4 +83,12 @@
         directory="./upgrades/select2js"
         />
 
+    <browser:page
+        for="z3c.form.interfaces.ISelectWidget"
```

Comment:
@jone Since the keywordwidget works for all ISelectWidget I chose this interface.
AND the Keywordwidget does not implement any interface so far. If a ISelectWidget is used it's generally also possible to use the keywordwidget.